practices/cloud-access.md

```diff
@@ -34,9 +34,28 @@ The canonical list of GCP projects we have access to is maintained
 
 ## AWS Access
 
-Each 2i2c engineer should get an individual [IAM User Account](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html)
-in every AWS account we have access to. This should have the broadest
+We have two ways to access AWS accounts.
+
+### AWS SSO
+
+For accounts we fully control, we use [AWS SSO](https://aws.amazon.com/single-sign-on/)
+to access them. We have an [AWS organization](https://aws.amazon.com/organizations/)
+set up, and sub-accounts for reach AWS account. This lets us use just
+*one* set of user credentials for multiple AWS accounts, with
```

Comment:
Does this mean that each of us can have multiple AWS accounts with a single log-in portal, or that 2i2c can have us all log into the account using the same credentials?